spec/acceptance_tests

```diff
@@ -1,5 +1,4 @@
 trap "kill 0" EXIT
 
-ruby spec/test_data.rb
```

Comment:
Originally, I create some new inboxes for `/inbox/12345`, `/inbox`, and `/inbox/guest-inbox` in `spec/test_data.rb`, and always load these test data before running acceptance test.

Reply:
I know, don't worry. I'll get these fix tonight. This is still a draft PR

Reply:
Ahh, I removed this because I didn't realize what it was actually doing. I move the setup into the tests themselves.